src/Model.java

```diff
@@ -1,4 +1,26 @@
 
 public class Model {
-
+	/* double array board 
+	 * 		[][][]			[1][2][3]
+	 *		[][][]			[4][5][6]
+	 *		[][][]			[7][8][9]
+	 *
+	 */
+	int[][] board;
+	int numOfMoves;
+	Player x;
+	Player o;

```

Comment:
Make all of the fields ``private final``, unless you really needed them to be packaged protected.